android/templates/build/root.build.gradle

```
@@ -10,9 +10,19 @@ buildscript {
 		classpath 'com.android.tools.build:gradle:7.0.4'
 		classpath 'com.google.gms:google-services:4.3.10'
 		classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+		<% for(var i=0; i<classpathes.length; i++) {%>
```

Minor change for better readability:
```suggestion
		<% for (let i = 0; i < classpathes.length; i++) {%>
```